### PR TITLE
feat: change dependency module version and sdk release version

### PR DESCRIPTION
**Pull request by gyuguen**
Invalid release version chaged.
Change the version accociated with `com.google.protobuf:protobuf-java:3.7.1`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 TBD
 
-## [v2.0.0-alpha.1](https://github.com/medibloc/panacea-java/releases/tag/v2.0.0-alpha.1) - 2021-07-01 
+## [v2.0.0-alpha.2](https://github.com/medibloc/panacea-java/releases/tag/v2.0.0-alpha.2) - 2021-07-01
```
Comment (gyuguen) [inline CHANGELOG.md:7]: If the alpha version changes, I'm thinking about whether to specify it together or just one.

File: build.gradle [hunk 2]
```diff
@@ -51,7 +51,7 @@ idea {
 }
 
 group 'org.medibloc.panacea'
-version '2.0.0'
+version '2.0.0-alpha.2'
```
Comment (gyuguen) [inline build.gradle:54]: It is valid version.

File: build.gradle [hunk 3]
```diff
@@ -61,9 +61,8 @@ repositories {
 }
 
 dependencies {
-    implementation 'org.bitcoinj:bitcoinj-core:0.14.7'
+    implementation 'org.bitcoinj:bitcoinj-core:0.15.10'
```
Comment (gyuguen) [inline build.gradle:64]: This dependency was also related to protobuf.

File: build.gradle [hunk 4]
```diff
@@ -61,9 +61,8 @@ repositories {
...
     implementation 'org.apache.commons:commons-lang3:3.12.0'
-    implementation 'com.google.protobuf:protobuf-java:3.6.1'
```
Comment (gyuguen) [inline build.gradle:66]: This project used to `com.google.protobuf:protobuf-java:3.7.1` version

File: build.gradle [hunk 5]
```diff
@@ -91,6 +93,7 @@ publishing {
                 password = System.getenv("GPR_API_KEY")
             }
         }
+//        mavenLocal()
```
Comment (gyuguen) [inline build.gradle:96]: If you deploy local repository, then use it.

File: src/main/java/org/medibloc/panacea/Wallet.java [hunk 6]
```diff
@@ -1,13 +1,12 @@
...
 
 import com.google.protobuf.ByteString;
-import com.subgraph.orchid.encoders.Hex;
 import cosmos.auth.v1beta1.BaseAccount;
 import cosmos.crypto.secp256k1.Keys;
 import lombok.Getter;
 import lombok.Setter;
 import org.apache.commons.net.util.Base64;
-import org.medibloc.panacea.encoding.Bech32;
+import org.bouncycastle.util.encoders.Hex;
```
Comment (gyuguen) [inline src/main/java/org/medibloc/panacea/Wallet.java:9]: It is associated to `bitcoinj-core`

File: CHANGELOG.md (hunk 1 above)
Comment (youngjoon-lee) [reply CHANGELOG.md:7]: I think your strategy is good enough (replacing alpha.1 with alpha.2 in CHANGELOG). 
In the end, we will replace it with v2.0.0. 
So, please accumulate all PR titles. 
(i think this PR title needs to be added to the `Bug fixes` section.)

Comment (gyuguen) [reply CHANGELOG.md:7]: Oh Thank you. I writed bug docs to CHANGELOG.MD.

**Review by youngjoon-lee — APPROVED** [review]
LGTM! Thanks for resolving this tricky dependency issue.

**Pull request merged**